query-graphs/lib/hyper.js

```diff
@@ -307,3 +300,15 @@ export function loadHyperPlan(graphString, graphCollapse) {
     var crosslinks = addCrosslinks(root);
     return {root: root, crosslinks: crosslinks, properties: properties};
 }
+
+// Load a JSON tree from text
+export function loadHyperPlanFromText(graphString, graphCollapse) {
```

Comment:
Refactoring seems fine. Does it have any functional impact?

Reply:
This refactoring here should not have any change in behavior for the "standalone-server".
When integrating query-graphs with JupyterLab, JupyterLab already provides the query plans as JSON. Hence, I needed a way to read plans from already parsed JSON and that's why I split the function into two.